src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const expense = expensesService.getExpenseById(id);
+
+  if (!expense) {
```

Comment:
The same applies here. If the expense doesn't exist, you could return a 'Expense not found' message instead of just a status code. Providing more specific error messages will make it easier for the client to understand what went wrong.